scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -37,8 +38,12 @@
 
 logger = logging.setup_logger(__file__, args.verbose, args.noColorLogger)
 
+print('Args add helicity given ?', args.addHelicityHistos)
```

Comment:
I would suggest making it logger.info or debug depending on your target. Also when you print the new filterProcs